dataland-backend/src/main/kotlin/org/dataland/datalandbackend/utils/JsonExampleFormattingConstants.kt

```diff
@@ -44,4 +44,16 @@ object JsonExampleFormattingConstants {
         "\"Services\": $procurementCategory," +
         "\"RawMaterials\": $procurementCategory" +
         "}"
+
```

Comment:
Nur aus Interesse. Ist das hier einfach eine Art Übersetzungs File für unsere jsons?

Reply:
Ich denke in dem File hier sind für einige Felder verschiedener Frameworks Beispiel-Jsons händisch abgetippt worden, die dann in dem "Example"-block landen, wenn du in der swagger-UI auf "TRy it out" klickst.

Wie das Code-seitig passiert siehst du, wenn du im gesamten Backend nach "JsonExampleFormattingConstans" sucht.

Passiert über Schema Annotations.